main.py

```diff
@@ -29,27 +29,33 @@ def __init__(self):
 
         # Used when moving two frames
         self.selected_frame = None
+        self.fullscreen = False
 
     def setup_ui(self):
         """Loads the main.ui file and sets up the window and grid."""
-        self.ui = uic.loadUi("ui/main.ui")
-        self.grid = VideoFrameGrid()
+        self.ui = uic.loadUi("ui/main.ui", self)
+        self.grid = VideoFrameGrid(self)
 
         self.container = self.ui.findChild(QtCore.QObject, "container")
         self.container.addLayout(self.grid)
+        self.menubar = self.ui.findChild(QtCore.QObject, "MenuBar")
 
         # Connect up all actions.
         self.actions = {}
         self.actions[MUTE_CHECKBOX] = self.ui.findChild(QtCore.QObject, MUTE_ALL_STREAMS)
         self.actions[MUTE_CHECKBOX].toggled.connect(self.mute_all_streams)
 
         self.ui.findChild(QtCore.QObject, EXPORT_STREAMS_TO_CLIPBOARD) \
-            .triggered.connect(self.export_streams_to_clipboard)
```

Comment:
I think `export_streams_to_clipboard` is clearer, especially in combination with the signal `EXPORT_STREAMS_TO_CLIPBOARD`. Up to you though.